tutorials/math/PRBS.C

```diff
@@ -0,0 +1,50 @@
+/// \file
+/// \ingroup tutorial_math
+/// \notebook -nodraw
+/// Tutorial illustrating the use of ROOT::Math::LFSR::GenerateSequence
+/// can be run with:
+///
+/// ~~~{.cpp}
+/// root > .x PRBS.C
+/// root > .x PRBS.C+ with ACLIC
+/// ~~~
+///
+/// These values match the PRBS sequences generated by a Keysight waveform generator
+/// except for the N first bits that appear at the end rather than at the beginning
+/// \see https://www.sciencedirect.com/science/article/pii/S0168900222002431#fig6
+///
+/// \macro_output
+/// \macro_code
+///
+/// \author Fernando Hueso-González
+
+#include <Math/LFSR.h>
+#include <iostream>
+
+void PRBS ()
+{
+   printf("\nLFSR::GenerateSequence PRBS3, PRBS4 and PRBS5 tests\n");
+   printf("==========================\n");
+
+   //PRBS3
+   std::array<uint16_t, 2> taps3 = {2, 3}; // Exponents of the monic polynomial
+   auto prbs3 = ROOT::Math::LFSR::GenerateSequence(std::bitset<3>().flip(), taps3);// Start value all high
```

Comment:
```suggestion
   std::vector<bool> prbs3 = ROOT::Math::LFSR::GenerateSequence(std::bitset<3>().flip(), taps3);// Start value all high
```
At least in the tutorial, I would spell out the types for more clarity.